.homeycompose/capabilities/measure_ozone.json

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "enum",
+  "title": {
+    "en": "Ozone",
+    "nl": "Ozone"
+  },
+  "getable": true,
+  "setable": false,
+  "uiComponent": "sensor",
+  "icon": "assets/icons/cloud-fog2-fill.svg",
+  "values": [
+    {
+      "id": "ON",
+      "title": {
+        "en": "On",
+        "nl": "Aan"
+      }
+    },
+    {
+      "id": "OFF",
+      "title": {
+        "en": "Off",
+        "nl": "Uit"
+      }
+    },
+    {
+      "id": "WAITING",
```

Comment:
I wonder if there is this value in availableValues for the component, but it doesnät really change it :)